analyses/create-subset-files/01-get_biospecimen_identifiers.R

```diff
@@ -185,7 +185,18 @@ files_to_subset <-
   files_to_subset[which(!(files_to_subset %in% c(polya_rsem_files[-1],
                                                  stranded_rsem_files[-1])))]
 
-# TODO: REMOVE THIS once you are no longer testing locally!
+# drop recurrently fused genes by histology file -- it is small enough to
+# include the entire thing
+files_to_subset <-
+  files_to_subset[-grep("fused-genes-byhistology", files_to_subset)]
```

Comment:
Hard to tell without running it but I think you could also make this this way: 
```suggestion
grep("fused-genes-byhistology", files_to_subset, invert = TRUE, value = TRUE) 
```